spec/integration/dry/configurable/setting_spec.rb

```diff
@@ -225,14 +234,24 @@
         expect(subclass.settings).to eql(Set[:db])
       end
 
-      it "configured parent copies config to the child" do
+      specify "configuring the parent before subclassing copies the config to the child" do
         klass.setting :db
 
         object.config.db = "mariadb"
 
         expect(subclass.config.db).to eql("mariadb")
       end
 
+      specify "configuring the parent after subclassing does not copy the config to the child" do
+        klass.setting :db
+
+        subclass = Class.new(klass)
+
+        object.config.db = "mariadb"
+
+        expect(subclass.config.db).to be nil
+      end
```

Comment:
As mentioned in the PR description, this is the only other behavior change introduced here, but I think it's a reasonable one.